tensorflow/tensorflow_plugin.cc

```
@@ -927,6 +973,10 @@ struct TensorflowGraphBase: public Function {
                 break;
             }
         }
+        else if (val.isEmbedding()) {
+            // TODO: infer consistent type for tensor, and convert it
```

do we need to throw an exception until this is done?